packages/core/src/application.ts

```diff
@@ -51,6 +57,7 @@ export class Application extends Context {
     if (!options) options = {};
 
     this.bind('http.port').to(options.http ? options.http.port : 3000);
+    this.api({basePath: '/', paths: {}});
```

Comment:
I'm +1 for app.rest, but gonna wait for @ritch's shotcall on the name. I'm also ok with deprecating into app.rest later on if necessary so you can move forward.